modules/apps/portal-cache/portal-cache-ehcache-impl/src/main/resources/ehcache/liferay-multi-vm.xml

```diff
@@ -219,23 +219,23 @@
 	<cache
 		eternal="false"
 		maxElementsInMemory="100000"
-		name="com.liferay.portal.kernel.dao.orm.FinderCache.com.liferay.portlet.expando.model.impl.ExpandoValueImpl"
+		name="com.liferay.portal.kernel.dao.orm.FinderCache.com.liferay.expando.model.impl.ExpandoValueImpl"
 		overflowToDisk="false"
 		timeToIdleSeconds="600"
 	>
 	</cache>
 	<cache
 		eternal="false"
 		maxElementsInMemory="100000"
-		name="com.liferay.portal.kernel.dao.orm.FinderCache.com.liferay.portlet.expando.model.impl.ExpandoValueImpl.List1"
+		name="com.liferay.portal.kernel.dao.orm.FinderCache.com.liferay.expando.model.impl.ExpandoValueImpl.List1"
 		overflowToDisk="false"
 		timeToIdleSeconds="600"
 	>
 	</cache>
 	<cache
 		eternal="false"
 		maxElementsInMemory="100000"
-		name="com.liferay.portal.kernel.dao.orm.FinderCache.com.liferay.portlet.expando.model.impl.ExpandoValueImpl.List2"
+		name="com.liferay.portal.kernel.dao.orm.FinderCache.com.liferay.expando.model.impl.ExpandoValueImpl.List2"
 		overflowToDisk="false"
```

Comment:
@slnn, the file liferay-multi-vm.xml seems defined cache for core. After moving Expando to module, please add them to module-multi-vm.xml in the service module. Please see https://help.liferay.com/hc/en-us/articles/360035581451-Introduction-to-Cache-Configuration

Reply:
Hi @yuhai 

From  liferay-multi-vm.xml, I can see this comment `"Caches in this file can be clustered and are configured to be clustered."`, so it is safe to move to module-multi-vm.xml in the service module?


Lily

Reply:
As we talked, I will move them into expano-service module